src/exportJarSteps/ResolveJavaProjectExecutor.ts

```diff
@@ -18,21 +18,29 @@ export class ResolveJavaProjectExecutor implements IExportJarStepExecutor {
     }
 
     public async execute(stepMetadata: IStepMetadata): Promise<ExportJarStep> {
-        await this.resolveJavaProject(stepMetadata, stepMetadata.entry);
+        await this.resolveJavaProject(stepMetadata);
         return this.getNextStep();
     }
 
-    private async resolveJavaProject(stepMetadata: IStepMetadata, node?: INodeData): Promise<void> {
-        if (node instanceof WorkspaceNode) {
-            stepMetadata.workspaceUri = Uri.parse(node.uri);
-            stepMetadata.projectList = await Jdtls.getProjects(node.uri);
+    private async setWorkspaceFolder(stepMetadata: IStepMetadata, uri: Uri): Promise<void> {
```

Comment:
The method name is confusing. Because it's not only set the workspace folder but also the project uri